### DISCUSSION TITLE
Choose a more readable font

### DISCUSSION THREAD

**Discussion by zaphus** [Tumpa-Prizrak/Tournament-Engine#2] [Ideas]
The current font, while cool and retro, has characters touching each other even at full screen, making it hard to read - and much harder to read when streaming with 4 games on the screen at the same time.
See if there is a more readable font that can survive being on a smaller window as people see it on Discord

Screenshot is from my 1440p screen, Discord maximised, stream at default size (so the way most people would see it - this or smaller)

![Screenshot 2023-08-16 201702](https://github.com/Tumpa-Prizrak/Tournament-Engine/assets/391294/881b60d2-bc3c-4a17-b6aa-b9b0f05e7e62)

**Comment by Tumpa-Prizrak** [comment]
It was a problem with OS. Resolved
